chore(docs): update securing pages tutorial (#3982)
nextauthjs/next-auth: docs/docs/tutorials/securing-pages-and-api-routes.md

}
```

### Next.js (Middleware)

With NextAuth.js 4.2.0 and Next.js 12, you can now protect your pages via the middleware pattern more easily. If you would like to protect all pages, you can create a `_middleware.js` file in your root `pages` directory which looks like this.

```js title="/pages/_middleware.js"
export { default } from "next-auth/middleware"
```

Otherwise, if you only want to protect a subset of pages, you could put it in a subdirectory as well, for example in `/pages/admin/_middleware.js` would protect all pages under `/admin`.

> This new `withAuth` middleware client only supports the `"jwt"` [session strategy](https://next-auth.js.org/configuration/options#session) at the moment.

Reviewer: ```suggestion
For the time being, the `withAuth` middleware only supports `"jwt"` as [session strategy](https://next-auth.js.org/configuration/options#session).
```
